### ISSUE TITLE
Error occurred updating bot with new SL/TP values: Other error occurred: record_invalid Invalid parameters {'trailing_enabled': ['is not included in the list']}.

### ISSUE THREAD

**Issue by kverpoorten** [cyberjunky/3commas-cyber-bots#98]
**Describe the bug**
I'm getting the following error from time to time (not always) with the trailingstoploss_tp.py script:

"3CQSBOT_MULTI_BANSHEE": BUSD_ATOM/1331030122 profit (3.02%) above activation (2.5%). StopLoss activated on 2.52%.

TakeProfit increased from 5.0% to 5.26%

Error occurred updating bot with new SL/TP values: Other error occurred: record_invalid Invalid parameters {'trailing_enabled': ['is not included in the list']}.

**To Reproduce**
not sure, it seems random... most of the time it works fine, sometimes I get this error instead and no SL is set

**Environment:**
 - Python version: python3 -V: 
Python 3.8.10
 - Operating System: uname -a
 Linux ubuntu-3commas-bots 5.4.0-97-generic #110-Ubuntu SMP Thu Jan 13 18:22:13 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
 - Version of software [e.g. 0.6.1]:
  0.9.0

**Additional context**

**Comment by cyberjunky** [comment]
Can you see with which update you get this, like Stoploss change or TP change?

**Comment by kverpoorten** [comment]
Yes, TP change.

The weird thing is I ran the script for several days with only one bot, no problems.

Yesterday I added a second bot (by adding a second bot ID to the array), and with the first attempt to update a deal from the second bot it happened.

Also one update did work fine on that same bot for some reason a little later.

I then created a complete new section in the ini file for that second bot, and it still went wrong 2 more times.

I now have the script running with only the second bot (removed the first one) and it's been running fine since then.

So either that was a complete coincidence and 3commas was having problems only while I had 2 bots in the config, or it only happens when there are more than one bot configured...

Sorry I can't be more help, I've found no good way to consistently reproduce it.
